keps/sig-apps/592-ttl-after-finish/README.md

```diff
@@ -250,17 +219,194 @@ Mitigations:
-not, and when it satisfies users' need for cleaning up finished resource
-objects, without regressions.
+### Alpha
+
+ - For alpha graduation, the feature implemented for Job, as future work it can be extended to Pods, but that should happen under a separate feature flag.
+ - Unit and e2e tests
+
+### Alpha -> Beta
+
+- Appropriate metrics are agreed on and implemented
+- upgrade/rollback manually tested 
 
-This will be promoted to GA once it's gone a sufficient amount of time as beta
-with no changes. 
+### Beta -> GA
+
+- Make a decision on wehther or not the feature should be extended to pods
+- Enabled in Beta for at least two releases without complaints
 
 [umbrella issues]: https://github.com/kubernetes/kubernetes/issues/42752
 
-## Implementation History
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [x] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name: TTLAfterFinished
+    - Components depending on the feature gate: kube-apiserver, kube-controller-manager
+  - [ ] Other
+    - Describe the mechanism:
+    - Will enabling / disabling the feature require downtime of the control
+      plane?
+    - Will enabling / disabling the feature require downtime or reprovisioning
+      of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled).
+
+* **Does enabling the feature change any default behavior?**
+  No.
+
+* **Can the feature be disabled once it has been enabled (i.e. can we roll back
+  the enablement)?**
+  Yes. One caveat here is that Jobs created with TTLSecondsAfterFinished set when 
+  the feature was enabled will continue to have that field set when the feature is disabled,
+  but will not have any effect.
+
+* **What happens if we reenable the feature if it was previously rolled back?**
+  It should work as expected.
+
+* **Are there any tests for feature enablement/disablement?**
+  No.
+
+### Rollout, Upgrade and Rollback Planning
+
+* **How can a rollout fail? Can it impact already running workloads?**
+  It shouldn't impact already running workloads. This is an opt-in feature since users need to 
+  explicitly set the TTLSecondsAfterFinished parameter in the job spec, which is silently dropped
+  by the api-server if the feature is disabled.
+
+* **What specific metrics should inform a rollback?**
+  Unexpected restarts of kube-controller-manager
+
+* **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
+  Manually tested.
+
+* **Is the rollout accompanied by any deprecations and/or removals of features, APIs, 
+fields of API types, flags, etc.?**
+  No
+
+### Monitoring Requirements
+
+_This section must be completed when targeting beta graduation to a release._
+
+* **How can an operator determine if the feature is in use by workloads?**
+  - The `workqueue_adds_total` tracks the number of finished Jobs with ttlSecondsAfterFinished set.
```

Comment:
I think you need to provide some label to that metric (controller name or sth like that), right?
Can you please make that explicit?